### PR TITLE
Bratseth/avoid schema dir

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/searchdefinition/ApplicationBuilder.java [hunk 1]
```diff
@@ -201,28 +195,28 @@ public Schema add(Schema schema) {
...
             throw new ParseException(stream.formatException(Exceptions.toMessageString(pe)));
         }
     }
 
-    private void addRankProfileFiles(Schema schema, String schemaPath) {
-        if (applicationPackage == null || schemaPath == null) return;
-        Path rankProfilePath = Path.fromString(schemaPath).append(schema.getName());
+    private void addRankProfileFiles(Schema schema) {
+        if (applicationPackage == null) return;
+        Path rankProfilePath = ApplicationPackage.SCHEMAS_DIR.append(schema.getName());
```
Comment (hmusum) [inline config-model/src/main/java/com/yahoo/searchdefinition/ApplicationBuilder.java:219]: Will not work with old schema dir (ApplicationPackage.SEARCH_DEFINITIONS_DIR), is that intended?

**Review by hmusum — APPROVED** [review]
One comment, LGTM

File: config-model/src/main/java/com/yahoo/searchdefinition/ApplicationBuilder.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/main/java/com/yahoo/searchdefinition/ApplicationBuilder.java:222]: Yes, but maybe it'll just cause confusion. I added it.

**Comment by bratseth** [comment]
... adding that broke some tests.

**Pull request merged**